charts/acend-training-chart/values.yaml

```diff
@@ -36,11 +36,11 @@ acendTraining:
       podAnnotations: {}
       ingress:
         annotations:
-          kubernetes.io/ingress.class: nginx-public
           kubernetes.io/tls-acme: "true"
+        ingressClassName: nginx-public
         useDefaultSecret: true
         appname: hugo-training-template
-        domain: k8s.puzzle.ch
+        domain: puzzle.ch
```

Comment:
default is `training.acend.ch`, can you change it to this.

Reply:
Is changed, thanks.